res/values-zh-rCN/arrays.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2022 FlamingoOS Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<resources>
+
+    <string-array name="allowlist_hide_background_in_battery_usage" translatable="false">
+        <item>com.google.android.GoogleCamera</item>
+    </string-array>
+
+    <!-- App lock timeout -->
+    <string-array name="app_lock_timeout_entries">
```

Comment:
Remove all other arrays except this one, only translated arrays should be present here ideally, others will be mapped from default flamingo_arrays.xml file